src/components/main/note/final-view/FinalViewFooter.js

```diff
@@ -35,11 +22,11 @@ const FinalViewFooter = ({ userId }) => {
 };
 
 FinalViewFooter.propTypes = {
-  userId: PropTypes.string,
+  userName: PropTypes.string,
 };
 
 FinalViewFooter.defaultProps = {
-  userId: null,
+  userName: 'Anonymous',
```

Comment:
Factor out? Maybe translate, but you'll need to define the default value within the component.